src/index.js

```diff
@@ -29,35 +32,43 @@ const Beachball = {
       scope,
       baseUrl,
       storage_key,
+      hostUrl,
+      sdkHostUrl,
     })
 
     getLoginStatus().then(saveSession)
     getLoginFromHash()
       .then(saveSession)
       .then(trimHash)
       .catch(() => {})
+
+    if (
```

Comment:
This if statement can just be on one line. Since it's not checking multiple things